src/java/org/apache/cassandra/locator/Endpoints.java

```diff
@@ -85,17 +84,68 @@ public boolean contains(Replica replica)
         return Collections.unmodifiableMap(byEndpoint);
     }
 
-    // TODO: TR-Review ignoreConflicts is not an acceptable solution here - we need to explicitly resolve them in case of transient/full mismatch
-    public static <E extends Endpoints<E>> E concat(E natural, E pending, Conflict ignoreConflicts)
+    public E withoutSelf()
```

Comment:
The first time I saw self it was pretty surprising. Never heard broadcast address referred to that way before.